website/docs/r/private_endpoint_regional_mode.html.markdown

```diff
@@ -0,0 +1,137 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: private_endpoint_regional_mode"
+sidebar_current: "docs-mongodbatlas-resource-private_endpoint_regional_mode"
+description: |-
+    Provides a Private Endpoint Regional Mode resource
+---
+
+# private_endpoint_regional_mode
+
+`mongodbatlas_private_endpoint_regional_mode` provides a Private Endpoint Regional Mode resource. This represents a the regionalized private endpoint setting for a Project. Enable it to allow region specific private endpoints.
+
+~> **IMPORTANT:**You must have one of the following roles to successfully handle the resource:
+  * Organization Owner
+  * Project Owner
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.
+
+
```

Comment:
I would add the parameter warning here: 

```suggestion
~> **WARNING:**Your [connection strings](https://www.mongodb.com/docs/atlas/reference/faq/connection-changes/#std-label-connstring-privatelink) to existing multi-region and global sharded clusters change when you enable this setting.  You must update your applications to use the new connection strings. This might cause downtime.

```